pkgs/applications/misc/warpd/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, libXi
+, libXinerama
+, libXft
+, libXfixes
+, libXtst
+, libX11
+, libXext
+, waylandSupport ? false, cairo, libxkbcommon, wayland
```

Comment:
Are you sure that libxkbcommon is a dependency of Wayland here?

Reply:
I figure, reasoning being here:

https://github.com/rvaiya/warpd/blob/master/src/platform/wayland/wayland.h#L22

The project's README states xkbcommon as a dependency (for an ubuntu build), and libxkbcommon seemed to, based on a best guess, fill that hole.

Reply:
Alright, fair enough.